setup.cfg

```diff
@@ -1,13 +1,19 @@
-[nosetests]
-nocapture=1
-cover-package=zope.container
-cover-erase=1
-with-doctest=0
-where=src
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/c-code
+[bdist_wheel]
+universal = 1
```

Comment:
Is universal wheel reasonable here? I thought it actually contains c-code.

Reply:
Yes it is reasonable, we do this for all repositories containing C code.